src/Equinor.ProCoSys.IPO.Query/GetInvitationById/GetInvitationByIdQueryHandler.cs

```diff
@@ -226,6 +226,7 @@ private IEnumerable<ParticipantDto> ConvertToParticipantDto(IReadOnlyCollection<
                         participant.Note,
                         participant.Attended,
                         IsSigningParticipant(participant) && CurrentUserCanSignAsPersonInFunctionalRole(participant).Result,
+                        IsSigningParticipant(participant) && CurrentUserCanSignAsPersonInFunctionalRole(participant).Result,
```

Comment:
Also, I think we should do some more checks before setting the IsSigner-flag. I think maybe we should have the same checks that we validate against in the signingEndpoint. In that case then it just means throwing on a check that the invitation is not cancelled.